src/components/resource-picker/files-upload-screen.yml

```diff
@@ -14,14 +14,17 @@ preprocessed:
   en: edited
   de: bearbeitet
 stage_uploadNotStarted:
-  en: "The following {fileCount, plural, =1 {file} other {# files}} will be uploaded. You can edit images before upload by clicking on the pen icon."
-  de: "Die {fileCount, plural, =1 {folgende Datei wird} other {folgenden # Dateien werden}} hochgeladen. Du kannst Bilder vor dem Upload durch Klick auf das Stift-Symbol bearbeiten."
+  en: "The following {fileCount, plural, =1 {file} other {# files}} will be uploaded.."
```

Comment:
There's a double period at the end here

Reply:
Only because I did not trust the upload process and wanted to leave room for error.
Fixed, thanks!

Reply:
:rofl: